config/locales/en.yml

```diff
@@ -967,6 +967,8 @@ en:
           shopfront_sort_by_producer: "By producer"
           shopfront_sort_by_category_placeholder: "Category"
           shopfront_sort_by_producer_placeholder: "Producer"
+          display_remaining_stock: "Display remaining stock in shopfront"
+          display_remaining_stock_tip: "Display remaining stock in shopfront"
```

Comment:
Why do we need a tooltip if it's not showing us any more information? Maybe we should explain a bit more here.